### PR TITLE
Merge pull request #7 from tenyiting/chapter_6_redo

### PR DESCRIPTION
**Pull request by tenyiting**
Chapter6 redo 
ready

### CODE REVIEW

**Comment by tenyiting** [comment]
ready

**Comment by tenyiting** [comment]
ready

**Comment by tenyiting** [comment]
ready

File: test/models/user_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (ghost) [inline test/models/user_test.rb]: please delete this file

File: test/fixtures/users.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (ghost) [inline test/fixtures/users.yml]: please delete this file

**Comment by tenyiting** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**